assembl/static2/js/app/components/administration/voteSession/customizeGaugeForm.jsx

```diff
@@ -46,7 +46,7 @@ type State = {
   saving: boolean
 };
 
-export class DumbCustomizeGaugeForm extends React.Component<DefaultProps, Props, State> {
+export class DumbCustomizeGaugeForm extends React.Component<Props, State> {
```

Comment:
You should remove DefaultProps type above now.

Reply:
I did